man/ostree-prepare-root.xml

```diff
@@ -113,6 +113,14 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
                 <term><varname>sysroot.readonly</varname></term>
                 <listitem><para>A boolean value; the default is false.  If this is set to <literal>true</literal>, then the <literal>/sysroot</literal> mount point is mounted read-only.</para></listitem>
           </varlistentry>
+            <varlistentry>
+                <term><varname>composefs.enable</varname></term>
+                <listitem><para>This can be yes, no or maybe, the default is no.  If this is set to <literal>yes</literal>, then composefs is always used, and the boot fails if it is not available. If set to <literal>maybe</literal>, then composefs is used if supported.</para></listitem>
+          </varlistentry>
+            <varlistentry>
+                <term><varname>composefs.keyfile</varname></term>
+                <listitem><para>Path to a Ed25519 public key in the initramfs. If this is specified, the boot fails unless the commit is signed with this key and the composefs image digest matches the one specified in the digest.</para></listitem>
```

Comment:
> composefs image digest matches the one specified in the digest.

composefs image digest matches the one embedded in the target OSTree commit metadata.

?